src/main/kotlin/car_race/logic/GameRound.kt

```diff
@@ -0,0 +1,28 @@
+package car_race.logic
+
+class GameRound private constructor(
+    val value: Long
+) {
+
+    companion object {
+        private const val INVALID_TYPE_MESSAGE = "게임 횟수는 양의 정수여야 합니다."
+
+        fun from(input: String): GameRound {
+            validate(input)
+
+            return GameRound(input.toLong())
+        }
+
+        private fun validate(input: String) {
+            try {
+                input.toLong()
+            } catch (e: NumberFormatException) {
+                throw IllegalArgumentException(
+                    INVALID_TYPE_MESSAGE,
+                    e
+                )
+            }
+            check(input.toLong() > 0) { throw IllegalArgumentException(INVALID_TYPE_MESSAGE) }
+        }
+    }
```

Comment:
`toLong()`은 성능에 큰 영향을 주지 않지만, 3번을 호출하는건 약간 아쉬움이 남네요!
코틀린에서는 `try`,  `if`의 결과값을 변수로 담아서 사용할 수 있어요.
```suggestion
        fun from(input: String): GameRound {
            return GameRound(toLong(input.toLong))
        }

        private fun validate(input: String): Long {
            val number = try {
                input.toLong()
            } catch (e: NumberFormatException) {
                throw IllegalArgumentException(
                    INVALID_TYPE_MESSAGE,
                    e
                )
            }
            require(number > 0) { INVALID_TYPE_MESSAGE }
        }
    }
```

Reply:
코드 감사합니다 주신 내용대로 수정해보겠습니다

Reply:
[refactor: [3단계] 코드리뷰 적용](https://github.com/next-step/kotlin-racingcar/pull/832/commits/73de392bccfff25713bc412061ec7527564244d3)
에 적용하였습니다